benchmarks/tosi_et_al_2015_gcubed/Tosi_Case2.prm

```diff
@@ -0,0 +1,194 @@
+# and the parameter controlling the temperature
+# dependence of viscosity "Thermal viscosity parameter"
+# is always 1e5.
+# Between the cases the "Pressure viscosity parameter" ($\Delta_{\eta}_z$),
+# the "Yield stress" ($\sigma_y$) and the "Nonlinear viscosity parameter" ($\eta^*$)
+# are varied, see Eq. (6-8) and Table 1 of the paper.
+subsection Material model
+  set Model name = TosiMaterial
+
+  subsection Tosi benchmark
+    set Reference density             = 1
+    set Reference specific heat       = 1
+    set Reference temperature         = 0
+    set Thermal conductivity          = 1
+    set Thermal expansion coefficient = 1e-06 
+    set Reference viscosity           = 1e-1
+    set Thermal viscosity parameter   = 1e5
+    set Pressure viscosity parameter  = 1
+    set Yield stress		      = 1
+    set Nonlinear viscosity constant  = 1e-3
+    set Initial viscosity	      = 0.155
+    set Minimum viscosity	      = 1e-6
+    set Maximum viscosity	      = 10
+  end
+end
+
+
+# The settings above all pertain to the description of the
+# continuous partial differential equations we want to solve.
+# The following section deals with the discretization of
+# this problem, namely the kind of mesh we want to compute
+# on. We here use a globally refined mesh without
+# adaptive mesh refinement.
+subsection Mesh refinement
+  set Initial global refinement                = 4
+  set Initial adaptive refinement              = 0
+  set Time steps between mesh refinement       = 0
+end
+
+
+# The final part is to specify what ASPECT should do with the
+# solution once computed at the end of every time step. The
+# process of evaluating the solution is called `postprocessing'
+# and we choose to compute velocity and temperature statistics,
+# statistics about the heat flux through the boundaries of the
+# domain, and to generate graphical output files for later
+# visualization. These output files are created every time
+# a time step crosses time points separated by 0.05. Given
+# our start time (0.0) and final time (1.0) this means that
+# we will obtain 20 output files.
+#
+# The statistics file includes the data reported in Tosi et al. (2015):
+# - average temperature (temperature statistics)
+# - top and bottom Nusselt numbers (heat flux statistics)
+# - Vrms over the whole domain (velocity statistics)
+# - Vrms along the surface (velocity boundary statistics)
+# - Vmax along the surface (velocity boundary statistics)
+# - average rate of viscous dissipation (TosiPostprocessor)
+# - average rate of work against gravity (TosiPostprocessor)
+# - percentage error difference work and dissipation (TosiPostprocessor)
+#
+# Tosi et al. (2015) also report depth averages; averages for viscosity
+# and temperature are produced every 0.05 units of time.
+subsection Postprocess
+  set List of postprocessors = velocity statistics, temperature statistics, heat flux statistics, visualization, TosiPostprocessor, depth average, velocity boundary statistics
```

Comment:
same here -- which, by the way, suggests that there is a certain amount of text in this file that is shared between this and the other .prm file. Have you thought externalizing the shared text into a separate "include file" that you can then include into each of the .prm files?